src/lib/plot/contour.typ

```diff
@@ -55,6 +76,8 @@
 
   let lerp(a, b) = {
     if a == b { return a }
+    else if a == none { return 1 }
+    else if b == none { return 0 }
```

Comment:
what happens if both are none? why can they even be none?

Reply:
Because the algorithm look outsides the data to make the edge detection at domain limits possible. This might change with the fix for the domain edges, but is correct for filling.